### PR TITLE
2480 - Run migrations on app start

### PR DESCRIPTION
**Pull request by sorja**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "run:dev:server": "ts-node-dev --exit-child src/server/start",
     "run:prod": "run-s build start",
     "run:prod:server": "node dist/server/start",
-    "start": "run:prod:server",
+    "start": "run-s migration-public:run run:prod:server",
```
Comment (minotogna) [inline package.json:27]: Also in dev mode ?

Comment (sorja) [reply package.json:27]: Sure

File: package.json [hunk 2]
```diff
@@ -21,10 +21,11 @@
     "migration-steps:watch": "ts-node-dev --respawn --transpile-only --exit-child --watch src/test/migrations/ src/test/migrations/ -- --watch",
     "run:dev": "run-p run:dev:client run:dev:server",
     "run:dev:client": "cross-env NODE_ENV=development webpack serve --config webpack.config.babel.js",
-    "run:dev:server": "ts-node-dev --exit-child src/server/start",
+    "dev:server": "ts-node-dev --exit-child src/server/start",
+    "run:dev:server": "run-s migration-public:run dev:server",
```
Comment (minotogna) [inline package.json]: Why double script ? One is enough ?

Comment (sorja) [reply package.json]: https://github.com/mysticatea/npm-run-all/blob/master/docs/run-s.md

`npm-run-all` expects NPM script as param, i tried different combinations, but all give error if not written as in this PR

One error for example when using quotation marks:

image

Comment (sorja) [reply package.json]: Would you prefer `&&` syntax over `run-s` then?

Comment (minotogna) [reply package.json]: Up to you

Comment (sorja) [reply package.json]: We can keep as it is in this PR then 👍

Comment (minotogna) [reply package.json]: Can we please have one script only ?

Comment (sorja) [reply package.json]: We cannot using the `npm-run-all`, so we must use `&&` which then defeats the purpose of having the package `npm-run-all`

Changed to `&&` to close PR

Comment (minotogna) [reply package.json]: let's talk about that later on. but the run-p command is still good to use?

**Review by minotogna — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (sorja) [reply package.json]: parallel (p) and sequential (s)
but only work on package.json scripts

**Pull request merged**